cmake/modules/AddSwift.cmake

```diff
@@ -223,7 +223,6 @@ function(_add_variant_swift_compile_flags
   list(APPEND result
       "-sdk" "${SWIFT_SDK_${sdk}_PATH}"
       "-target" "${SWIFT_SDK_${sdk}_ARCH_${arch}_TRIPLE}"
-      "-resource-dir" "${SWIFTLIB_DIR}")
```

Comment:
@jrose-apple I would like this in a separate commit. I think it is important to explain via the commit message why this change was important.